onnxscript/function_libs/torch_lib/registration.py

```diff
@@ -63,8 +69,27 @@ def items(self) -> Generator[tuple[str, OverloadedFunction], None, None]:
 default_registry = Registry()
 
 
+def _check_and_normalize_names(name: str | tuple[str, ...]) -> tuple[str, ...]:
+    names: tuple[str, ...]
+
+    if isinstance(name, str):
+        names = (name,)
+    else:
+        names = name
+    if not isinstance(names, tuple):
+        raise TypeError(f"Name must be a string or a tuple of strings, got {name}")
+    for name_ in names:
+        if name_.endswith(".default") or not _QUALIFIED_OPERATOR_NAME_REGEX.fullmatch(name_):
```

Comment:
so we don't allow registering for default? should we call out in `torch_op` docstring